file/provider/expand_volume_test.go

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2021 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package provider
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/models"
+	volumeServiceFakes "github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/vpcfilevolume/fakes"
+	"github.com/IBM/ibmcloud-volume-interface/lib/provider"
+	util "github.com/IBM/ibmcloud-volume-interface/lib/utils"
+	"github.com/IBM/ibmcloud-volume-interface/lib/utils/reasoncode"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+func TestExpandVolume(t *testing.T) {
+	logger, teardown := GetTestLogger(t)
+	defer teardown()
+
+	var (
+		fileShareService *volumeServiceFakes.FileShareService
+	)
+
+	testCases := []struct {
+		testCaseName       string
+		volumeID           string
+		baseVolume         *models.Share
+		newSize            int64
+		expectedErr        string
+		expectedSize       int64
+		expectedReasonCode string
+	}{
+		{
+			testCaseName: "OK",
+			volumeID:     "16f293bf-test-4bff-816f-e199c0c65db5",
```

Comment:
do we need `volumeID` can we get it from models.Share.ID?

Reply:
Need to give fake volumeID with valid format -> https://github.com/IBM/ibmcloud-volume-file-vpc/blob/master/file/provider/util.go#L306-L309 is checked as part of expand call